Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -848,42 +848,48 @@ void VideoInterfaceManager::EndField(FieldType field, u64 ticks)
 // Run when: When a frame is scanned (progressive/interlace)
 void VideoInterfaceManager::Update(u64 ticks)
 {
+  constexpr u32 odd_field_begin = 0;
+  const u32 even_field_begin = GetHalfLinesPerOddField();
```

Comment:
Actually, missed this originally, it might be good to add a comment here?  The fact the function is GetHalfLinesPer**Odd**Field but the variable is **even**_field_begin is strange and a bit counter-intuitive.

Reply:
(also if you look at this diff, everywhere uses `GetHalfLinesPerEvenField()` so one would wonder if this was a mistake)

Your comment in OP makes me think it wasn't.  I've looked at this file a handful of times but not enough to really be able to sign off on logic changes..

Reply:
I think awareness of even field coming after the odd field is heavily assumed throughout much of this file and the way I've established the variable names should be very clear now. I can add a comment `// Even field begins after the odd field` if you want, but it's maybe unnecessary?

Reply:
I think what you're saying is if we treat it like a container.  If there's 5 things in a vector.  Then querying the size is 5 but the next position will also be 5.

So `GetHalfLinesPerOddField()` will return a count of the half lines per an odd field but could also be the 'index' of the even fields.  Maybe?

If that's understood throughout the file, then let's just resolve this.  it just seemed a bit odd just looking at this code in isolation.

Reply:
Yeah, I'll add a comment though. :D